src/OpenstoreSchema/Core/Entity/ProductTranslation.php

```diff
@@ -95,6 +95,11 @@ class ProductTranslation
      */
     private $usp;
 
+    /**
+     * @ORM\Column(type="string", length=2000, nullable=true, options={"comment" = "Product marketing text"})
```

Comment:
@lgillie Good, let me check few things about length... Mysql have some limitations regarding how many chars, indexes... are allowed for a single table. Travis will help us too.

Reply:
Ok all good for the length checks... Travis have a problem right now installing Mariadb 10.3... that's why it fails... We can ignore.

https://github.com/belgattitude/openstore-schema-core/commit/9bd0989e04badee4078ce4231e351d8e3e48782c